packages/notification-service/src/firebase.ts

```diff
@@ -145,7 +145,7 @@ export function setLastBlockNotified(newBlock: number) {
   // we set it here ourselves to avoid race condition where we check for notifications
   // again before it syncs
   lastBlockNotified = newBlock
-  lastBlockRef.set(newBlock)
+  await lastBlockRef.set(newBlock)
```

Comment:
Ditto as above. Lets replace await with return here and make this function not async